crates/agglayer-types/src/lib.rs

```diff
@@ -105,7 +105,7 @@ pub enum Error {
     )]
     NullifierPathGenerationFailed {
         source: SmtError,
-        global_index: GlobalIndex,
+        global_index: U256,
```

Comment:
@iljakuklic such change are breaking for the serialization/deserialization iirc right? In that case should I just create a new variant with `U256` instead of `GlobalIndex`?

Reply:
> @iljakuklic such change is breaking for the serialization/deserialization iirc right? In that case should I just create a new variant with `U256` instead of `GlobalIndex`?

Yep, a new variant is required here.

Once we get the up-to-date `interop` dependency, I will be able to create a PR taking a snapshot of the structure of these types which will make it easier to spot the incompatibilities.